changelogs/fragments/module-zabbix_host-interfaces-not-required.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - zabbix_host - module will no longer require ``interfaces`` to be present when creating host  with Zabbix 5.2.
```

```suggestion
  - zabbix_host - module will no longer require ``interfaces`` to be present when creating host  with Zabbix 5.2 (https://github.com/ansible-collections/community.zabbix/pull/291).
```

Can you please add this PR URL to the end?
After that, please merge this PR.